source/upgrade.txt

```diff
@@ -61,6 +61,11 @@ Version 4.8 Breaking Changes
   earlier. To learn more about this change, see the :ref:`<java-server-release-change-v4.8>`
   section.
 
+- The ``RecordCodec``, implemented in v4.6, decoded ``List`` and ``Map``
+  fields as ``Document`` values instead of instances their corresponding POJO
+  or record classes. In this version, the ``RecordCodec`` decodes them to
```

Comment:
```suggestion
  fields as ``Document`` values instead of instances of their corresponding POJO
  or record classes. In this version, the ``RecordCodec`` decodes them to
```